Common/include/option_structure.hpp

```diff
@@ -1506,7 +1509,9 @@ static const map<string, ENUM_OUTPUT> Output_Map = CCreateMap<string, ENUM_OUTPU
 ("SURFACE_PARAVIEW", SURFACE_PARAVIEW_BINARY)
 ("RESTART_ASCII", RESTART_ASCII)
 ("RESTART", RESTART_BINARY)
-("CGNS", CGNS);
+("CGNS", CGNS)
+("STL", STL)
+("STL_BINARY", STL_BINARY);
```

Comment:
Can you please add an error check for STL_BINARY if there isn't one already in case someone tries to use this?

Reply:
Done